model/newpaper.js

```diff
@@ -17,6 +23,33 @@ exports.fetchArticleById = (articleId) => {
     });
 };
 
+// SELECT * FROM articles 
+//     WHERE article_id = $1;
+
+// exports.commentCountArticle = (article) => {
+//     const articleObj = article
+//     const articleId = articleObj.article_id
+//     return db.query(
+//         `SELECT COUNT(article_id) AS comment_count
+//         FROM comments
+//         INNER JOIN articles ON articles.article_id = comments.article_id;`)
+//         .then(({body}) => {
+//             return body
+//         });
+// };
+exports.commentCountArticle = (article) => {
+    const articleObj = article
+    const articleId = articleObj.article_id
+    return db.query(
+        `SELECT * FROM comments
+        WHERE article_id = $1;`, [articleId])
+        .then(({body}) => {
+            console.log(body)
+        })
+};
+
```

Comment:
Nicely done! get rid of the old code and commented out stuff, your current solution looks great :D